x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/configure_fields.tsx

```diff
@@ -68,109 +68,115 @@ export const ConfigureFields: React.FC = () => {
 
   return (
     <>
-      <EuiTitle size="xs">
-        <h4>
-          {i18n.translate(
-            'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.steps.fields.title',
-            { defaultMessage: 'Select field mappings' }
-          )}
-        </h4>
-      </EuiTitle>
-      <EuiSpacer size="m" />
-      <EuiText color="subdued">
-        <p>
-          {i18n.translate(
-            'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.steps.fields.description',
-            {
-              defaultMessage:
-                'Choose fields to be enhanced from your existing documents or manually enter in fields you anticipate using.',
-            }
-          )}
-        </p>
-      </EuiText>
-      <EuiSpacer />
-      <EuiForm component="form">
-        <EuiFlexGroup>
-          <EuiFlexItem>
-            <EuiFormRow
-              fullWidth
-              label={i18n.translate(
-                'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.steps.fields.sourceFieldLabel',
+      <EuiFlexGroup>
+        <EuiFlexItem grow={3}>
+          <EuiTitle size="s">
+            <h4>
+              {i18n.translate(
+                'xpack.enterpriseSearch.content.indices.pipelines.addInferencePipelineModal.steps.fields.title',
+                { defaultMessage: 'Select field mappings' }
+              )}
+            </h4>
+          </EuiTitle>
+        </EuiFlexItem>
+        <EuiFlexItem grow={7}>
+          <EuiText color="subdued" size="s">
```

Comment:
Do we have any conventions on attributes:
- keeping them inline vs breaking them into new lines
- order of attributes

Reply:
Try to keep them alphabetized (but not necessary). New lines when the line becomes too long (linter should do this for you automatically).